runtime/common/src/evm/mod.rs

```diff
@@ -112,19 +112,19 @@ impl<T: pallet_aura::Config<AuthorityId = AuraId>> FindAuthor<H160> for FindAuth
 
 /// Passthrough router deployed bytecode as of this state
 /// https://github.com/centrifuge/liquidity-pools/blob/3000a5d0db8dd5f502545914447abe50d5f6d2ba/test/integration/PassthroughAdapter.sol
-///
-/// NOTE: If the above file changes, this code needs to be adapted as follows:
-///  1. Update the `liquidity-pools` submodule to the latest desired state
-///  2. Build with `forge-build`
-///  3. Go to `./out/PassthroughAdapter.sol` and copy-paste the
-///     `deployedBytecode` here.
-///  4. Run tests and update mismatching hashes.
-///  5. On Development chain, you might also have to update the
-///     `evm.accountCodes` storage via raw writing.
-///
-/// Blake256 hash of the deployed passthrough router contract code as
-/// Encoded::encode(Vec<Code>):
-/// `0x283d01c648e109952e3120e8928a19614c5c694477c780920ac29a748f96babf`
+//
+// NOTE: If the above file changes, this code needs to be adapted as follows:
+//  1. Update the `liquidity-pools` submodule to the latest desired state
+//  2. Build with `forge-build`
+//  3. Go to `./out/PassthroughAdapter.sol` and copy-paste the
+//     `deployedBytecode` here.
+//  4. Run tests and update mismatching hashes.
+//  5. On Development chain, you might also have to update the
+//     `evm.accountCodes` storage via raw writing.
+//
+// Blake256 hash of the deployed passthrough router contract code as
+// Encoded::encode(Vec<Code>):
+// `0x283d01c648e109952e3120e8928a19614c5c694477c780920ac29a748f96babf`
```

Comment:
The `<` character makes docs fail in they are computed as docs instead of normal comments. 

Modified this to be a coder comment instead of a user docs.

Reply:
Ah sorry for merging back that error. This branch had an outdated version though.

Reply:
Probably I think just pushed something in between your work. Now should pass 🤞🏻